File: PRAK604-2510817210003-MuhammadRaffaAzisha.py
kode = input()
pesan = input()

jumlah_bintang = 0
jumlah_pagar = 0
hasil_simbol = ""

if len(kode) != len(pesan):
    print("Panjang kalimat berbeda, pesan palsu")
else:
    for i in range(len(kode)):
        if kode[i] == ' ' and pesan[i] == ' ':
            hasil_simbol += ' '
            continue 
        
        if kode[i] == pesan[i]:
            hasil_simbol += '*'
            jumlah_bintang += 1
        else:
            hasil_simbol += '#'
            jumlah_pagar += 1

    print(hasil_simbol)
    print(f"* = {jumlah_bintang}")
    print(f"# = {jumlah_pagar}")

    if jumlah_bintang >= jumlah_pagar:
        print("Pesan Asli")
    else:
        print("Pesan Palsu")

File: PRAK604-2510817210003-MuhammadRaffaAzisha.c
#include <stdio.h>
#include <string.h>

int main() {
    char kode[100];
    char pesan[100];
    int i, len1, len2;
    int bintang = 0;
    int pagar = 0;

    scanf("%[^\n]%*c", kode);
    scanf("%[^\n]%*c", pesan);

    len1 = strlen(kode);
    len2 = strlen(pesan);

    if (len1 != len2) {
        printf("Panjang kalimat berbeda, pesan palsu\n");
    } else {
        for (i = 0; i < len1; i++) {
            if (kode[i] == ' ' && pesan[i] == ' ') {
                printf(" ");
            } 
            else if (kode[i] == pesan[i]) {
                printf("*");
                bintang++;
            } 
            else {
                printf("#");
                pagar++;
            }
        }

        printf("\n* = %d", bintang);
        printf("\n# = %d", pagar);

        if (bintang >= pagar) {
            printf("\nPesan Asli\n");
        } else {
            printf("\nPesan Palsu\n");
        }
    }

    return 0;
}